.github/workflows/e2e.yaml

```diff
@@ -107,6 +107,7 @@ jobs:
         uses: dawidd6/action-download-artifact@v2
         with:
           github_token: ${{ secrets.SDK_TYPESCRIPT_ACTION_READ_TOKEN || secrets.GITHUB_TOKEN }}
+          workflow: 58374855
```

Comment:
Instead of using the workflow id it might be more explanatory if we used the workflow file name `test.yml`.

Reply:
ok, i hope it works